project/urls.py

```diff
@@ -17,6 +17,9 @@
     path('', include('apps.file.urls')),
     path('', include('apps.user.urls')),
 
+    # Examples
```

Comment:
Maybe add in this line here also

# !!! REMOVE ME IN PRODUCTION !!!

Reply:
then its simple delete directory and remove the two lines in production.



You also mentioned in code chat maybe adding a check for if env production post warning or something which I think is a good addition